ngraph/python/CMakeLists.txt

```diff
@@ -73,11 +74,22 @@ file(GLOB_RECURSE SOURCES src/pyngraph/*.cpp)
 pybind11_add_module(_${PROJECT_NAME} MODULE ${SOURCES})
 
 target_include_directories(_${PROJECT_NAME} PRIVATE "${CMAKE_CURRENT_SOURCE_DIR}/src")
-target_link_libraries(_${PROJECT_NAME} PRIVATE ngraph::ngraph)
+
+target_link_libraries(_${PROJECT_NAME} PRIVATE ngraph::ngraph ngraph::frontend_manager)
+
 if (TARGET ngraph::onnx_importer)
     add_dependencies(_${PROJECT_NAME} ngraph::onnx_importer)
 endif()
 
+if(NGRAPH_UNIT_TEST_ENABLE)
+    add_subdirectory(tests/mock/mock_py_ngraph_frontend)
+    add_dependencies(_${PROJECT_NAME} mock_py_ngraph_frontend)
+    set_target_properties(mock_py_ngraph_frontend PROPERTIES LIBRARY_OUTPUT_DIRECTORY ${LIBRARY_OUTPUT_DIRECTORY_BIN})
```

Comment:
Or maybe I am wrong, but after the line 36 you don't need to add additional properties to target.

Reply:
As discussed, we've decided to not rework existing code, but add ARCHIVE_OUTPUT_DIRECTORY (+ PDB + COMPILE_PDB) as target properties for mock frontend as well